src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Steps/EndToEndStepsBase.cs

```
@@ -178,6 +180,11 @@ protected void PopulateLearner(FM36Learner learner, Training training, List<OnPr
                     LearningDeliveryValues = new LearningDeliveryValues
                     {
                         LearnAimRef = training.AimReference,
+                        LearnDelInitialFundLineType = training.FundingLineType,
+                        StdCode = TestSession.Learners.First(x => x.LearnRefNumber == learner.LearnRefNumber).Course.StandardCode,
```

create a variable for TestSession.Learners.First(x => x.LearnRefNumber == learner.LearnRefNumber).Course

---

Could the Learners collection be empty?

---

If the learners collection is empty then fail fast here